about.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>The Collision Summit 2023</title>
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <header id="social-header">
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href=""><img src="images/social/twitter-16.png" alt="twitter"></a></li>
+          <li><a href=""><img src="images/social/facebook-16.png" alt="facebook"></a></li>
+          <li><a href="#logout">Logout</a></li>
+          <li><a href="#mypage"> My Page</a></li>
+          <li><a href="#Portuguese"> Portuguese</a></li>
+        </ul>
+      </nav>
+    </header>
+    <header>
+      <div class="logo">
+        <a href="index.html"
+          ><img src="images/Logo/logo.avif" alt="logo"
+        /></a>
+      </div>
+      <nav class="hamb-menu">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="#Contact"> News</a></li>
+        </ul>
+      </nav>
+      <button type="button" id="header-button">Collision Campaign</button>
+    </header>
+    <div class="popup-menu">
+      <div class="menu-button">
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+        <span class="hamburger"></span>
+      </div>
+      <ul>
+        <li>
+          <a href="index.html">Home</a>
+        </li>
+        <li><a href="about.html">About</a>
+        </li>
+        <li><a href="#Contact">News</a>
+        </li>
+      </ul>
+    </div>
+    <main class="main-content">
+      <section id="about-main">
+        <div>
+          <h1>"Hello! Tech World"</h1>
+          <h2>WEB Summit 2023</h2>
+          <hr class="solid gone" />
+          <p id="introduction">
+            Web Summit events have gathered half a million
+            people across Web Summit in Lisbon, Collision in
+            North America, and RISE in Asia since its
+            beginnings as a 150-person conference in Dublin
+            in 2009.
+            <span id="more-introduction"
+              >Web Summit’s mission has been to create
+              software that enables meaningful connections
+              between the CEOs, founders, investors, media,
+              politicians and cultural figureheads who are
+              reshaping the world.</span
+            >
+          </p>
+          <h3>
+            Please contact us per email for any further questions about the WEB SUMMIT 2023!
+          </h3>
+          <p id="email">info@torontosummit.com</p>
+        </div>
+      </section>
```

Comment:
Please ensure that the pages should look almost identical to the original design (the margins should be the same as in the original design and the fonts).

Your Implementation | Design
:---:|:---:
![image](https://user-images.githubusercontent.com/10332499/164413494-8723f791-4bfb-41b5-b436-8644da8c3ce0.png) | ![image](https://user-images.githubusercontent.com/10332499/164413819-0fcc2566-9275-41c8-bfed-68eeca2489b8.png)

Reply:
resolved